libs/gl-client-py/glclient/glclient.pyi

```diff
@@ -35,7 +34,6 @@ class Credentials:
 class SignerHandle:
     def shutdown(self) -> None: ...
 
-
```

Comment:
Nit: idiomatic python requires two blank lines between top-level declarations.

Reply:
Oh, this was done by the formater (black). Let me check if I have some ill config.